src/registry.rs

```diff
@@ -244,7 +240,9 @@ lazy_static! {
         let reg = Registry::default();
 
         // Register a default process collector.
-        bridge::register_default_process_collector(&reg).unwrap();
+        if cfg!(feature = "process") {
```

Comment:
you should probably only have either the function or the function call be conditionally compiled in. So either drop this conditional, or move the body of `register_default_process_collector` here and add `target_os = "linux"` to the condition.